### PR TITLE
airlineController-testovi

### PR DESCRIPTION
**Pull request by likizruskihromana**
Gotovi testovi za Airline controller

### CODE REVIEW

File: SIAirline/SIAirline.UnitTests/Controllers/AirlineControllerTests.cs [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+namespace SIAirline.UnitTests.Controllers
+{
+    [TestFixture]
+    public class AirlineControllerTests
+    {
+        private ApplicationDbContext _context;
+        private AirlineController _controller;
+        private readonly ImmutableList<Airline> _airlines;  
+        private readonly ImmutableList<Aircraft> _aircrafts;
+        public AirlineControllerTests(){
```
Comment (tanerbajrovic) [inline SIAirline/SIAirline.UnitTests/Controllers/AirlineControllerTests.cs:17]: We should add some whitespace to improve the formatting of the code. Maybe include blank lines between different test methods and whenever we use `#region` and `#endregion`:

```c#
#region Some method
// Blank line here
public void SomeMethod()
{
...
}
// Blank line here
#endregion
```

File: SIAirline/SIAirline.UnitTests/Controllers/AirlineControllerTests.cs [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+            // Gives access to Response/Request
+            _controller.ControllerContext = new ControllerContext{ HttpContext = new DefaultHttpContext() };
+        }
+        [TearDown]
+        public void TearDown(){
+             _context.Dispose();
+        }
+        #region GetAirline(int id) method
+        [Test]
+        public async Task GetAirline_ReturnsAirline_WhenAirlineExistss(){
```
Comment (tanerbajrovic) [inline SIAirline/SIAirline.UnitTests/Controllers/AirlineControllerTests.cs:73]: Small typo, should be `GetAirline_ReturnsAirline_WhenAirlineExists`.

File: SIAirline/SIAirline.UnitTests/Controllers/AirlineControllerTests.cs [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+            var notFoundResult = result.Result as NotFoundObjectResult;
+            Assert.IsNotNull(notFoundResult, "NotFoundResult is null");
+            var errorMessage = notFoundResult.Value;
+            Assert.IsNotNull(errorMessage, "Error message is null");
+            Assert.That(expectedErrorMessage, Is.EqualTo(errorMessage));
+        }
+        #endregion
+        #region GetAirlines(int pageNumber, int pageSize)
+        [Test]
+        public async Task GetAirlines_ReturnsListOfAirlines_WhenAirlinesExitis(){
```
Comment (tanerbajrovic) [inline SIAirline/SIAirline.UnitTests/Controllers/AirlineControllerTests.cs:100]: Small typo, should be `GetAirlines_ReturnsListOfAirlines_WhenAirlinesExist`.

**Review by tanerbajrovic — CHANGES_REQUESTED** [review]
Looks good overall, all tests pass. Great idea to include tests for `GetAirlines(int pageNumber, int pageSize)` that check the edge cases. I have some formatting suggestions to keep our code cohesive.

**Review by tanerbajrovic — APPROVED** [review]
